oct/cli/package/common_options.py

```diff
@@ -14,21 +14,29 @@ def package_options(func):
     :return: decorated CLI command
     """
     return option(
-        '--update/--upgrade',
-        '-d/-g',
-        'update_current_stage',
-        default=False,
-        help='Update stage or upgrade to next stage.  [default: upgrade]',
+        '--upgrade',
```

Comment:
The previous implementation was better since it did not require someone to know the exact flow of the stages but once that flow is non-linear it's hard to think about so yes `--stage` seems reasonable.